src/components/common/Drawer.js

```diff
@@ -42,6 +42,18 @@ class Drawer extends Component {
   render() {
     const { user } = this.props;
     const { role } = this.props;
+    let iconRole;
+    let iconText;
+    if (role === 'Gather' || role === 'Recolectar') {
+      iconRole = HistoryIcon;
+      iconText = strings.history;
+    } else if (role === 'Weigh' || role === 'Pesar') {
+      iconRole = PocketIcon;
+      iconText = strings.filterByRole;
+    } else {
+      iconRole = BaleIcon;
+      iconText = strings.filterByRole;
+    }
     return (
       <View style={styles.containerWrapper}>
         <View style={styles.topHalf}>
```

Comment:
I know this is a slip up from my part when reviewing #109, and this isnt your mistake, but styles shouldn't be here. Could you move them to the corresponding styles file?